coretex/cli/resources/update_script_template.sh

```diff
@@ -49,7 +49,13 @@ OUTPUT_DIR="$HOME/.config/coretex"
 mkdir -p "$OUTPUT_DIR"
 
 # Generate the output filename based on the current Unix timestamp
-OUTPUT_FILE="$OUTPUT_DIR/ctx_autoupdate.log"
+if [[ "$1" == "--force" ]]; then
```

Comment:
So the "--force" flag defines if Node should be started or if automatic update should run? I like this, but lets invest a bit more time into this sh script so that the "coretex node start" only checks if Node is already running and stops it and then runs this sh script with this flag.